gtsfm/bundle/bundle_adjustment.py

```diff
@@ -49,13 +50,17 @@ def __init__(self, output_reproj_error_thresh: float, robust_measurement_noise:
         self._robust_measurement_noise = robust_measurement_noise
 
     def run(
-        self, initial_data: GtsfmData, wTi_list_gt: Optional[List[Pose3]] = None
+        self,
+        initial_data: GtsfmData,
+        wTi_list_gt: Optional[List[Pose3]] = None,
+        cameras_gt: Optional[List[PinholeCameraCal3Bundler]] = None,
```

Comment:
Doesnt PinholeCamera have the pose? Why do we need both wTi_gt and cameras_gt?

Reply:
removed it. Thanks!